deepchem/trans/transformers.py

```diff
@@ -1225,3 +1225,44 @@ def gaussian_blur(self, sigma=0.2):
             sigma - std dev. of the gaussian distribution
     """
     return scipy.ndimage.gaussian_filter(self.Image, sigma)
+
+  def shift(self, width, height, mode='constant', order=3):
+    """Shifts the image
+        Parameters:
+          width - amount of width shift(positive values shift image right )
+          height - amount of height shift(positive values shift image lower)
+          mode - Points outside the boundaries of the input are filled according to the given mode
+          (‘constant’, ‘nearest’, ‘reflect’ or ‘wrap’). Default is ‘constant’
+          order - The order of the spline interpolation, default is 3. The order has to be in the range 0-5.
+          """
+    if len(self.Image.shape) == 2:
+      return scipy.ndimage.shift(
+          self.Image, [height, width], order=order, mode=mode)
+    if len(self.Image.shape == 3):
+      return scipy.ndimage.shift(
+          self.Image, [height, width, 0], order=order, mode=mode)
+
+  def gaussian_noise(self):
+    '''Adds gaussian noise to the image
+        '''
+
+    x = (self.Image - np.min(self.Image)) / (
+        np.max(self.Image) - np.min(self.Image))
+    x = x + np.random.normal(loc=0, scale=0.1, size=self.Image.shape)
```

Comment:
How about providing arguments so the user can specify the mean and variance instead of just hardcoding values?  You can still use these as default values.